stock_available_mrp/data/mrp_data.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
`noupdate="1"` could be removed at `<odoo>`level and after that remove `<data>`tag.